src/Kushy/SymbolLoader.cs

```diff
@@ -260,12 +260,13 @@ private async Task<GlobalState> AddReferencedDatabasesAsync(GlobalState globals,
                 if (cluster == null || cluster.IsOpen)
                 {
                     // check to see if this is an actual cluster and get all database names
-                    var databaseNames = await GetDatabaseNamesAsync(clusterRef.Cluster).ConfigureAwait(false);
+                    var databaseNames = await GetDatabaseNamesAsync(clusterRef.Cluster, true).ConfigureAwait(false);
```

Comment:
You should not default this behavior to throwing exceptions.  If you think you need this to throw, then you should add a throwOnError argument to the function and pass this down from the calling code.

Reply:
added a param.